docs/docs/sourcing-from-builder-io.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Sourcing from Builder.io
+---
+
+[Builder.io](https://builder.io) has all the benefits of a modern headless CMS platform, plus the power of an easy to learn drag and drop editor that enables everyone to edit more than just content. Builder + Gatsby empowers your entire team to create performant and fully customizable pages quickly.
+
+<img src="https://imgur.com/HjBWIbv.gif" alt="Editor example" />
+
+---
+
+## Quick Start
+
+### Fresh project:
+
+To start on a fresh project quickly, take a look at the [BuilderIO Gatsby starter](/starters/BuilderIO/gatsby-starter-builder/)
+
+```shell
+gatsby new my-builder-site https://github.com/BuilderIO/gatsby-starter-builder
```

Comment:
I followed this and ran into some interesting errors which were solved by updating the `@builder.io/gatsby` plugin. you might want to take a look at this .

Reply:
thanks @Ekwuno for reporting back,  can you elaborate? I wasn't able to reproduce on my end but I did notice that the package-lock.json is not copied when doing `gatsby new ...`

Reply:
adding yarn.lock here https://github.com/BuilderIO/gatsby-starter-builder/pull/3

Reply:
Initially running gatsby new my-builder-site https://github.com/BuilderIO/gatsby-starter-builder and following the instructions gave this output ![](https://res.cloudinary.com/ekwuno/image/upload/v1586253138/Screenshot_2020-04-07_at_10.48.59.png)

I did an `npm install` to add a package-lock.json and ran an `npm i @builder.io/gatsby` and got this ![](https://res.cloudinary.com/ekwuno/image/upload/v1586254618/Screenshot_2020-04-06_at_14.44.58.png).. 

using the `yarn.lock` or `package-lock.json` depends on what package manager you are using. The instructions on the starter works as expected now though. 

These things can be weird sometimes 😅